jobs/views.py

```diff
@@ -2,33 +2,91 @@
 from jobs.models import Job
 from django.urls import reverse_lazy
 from .forms import JobForm
+from django.shortcuts import redirect
 
 
 class JobView(ListView):
     model = Job
     template_name = 'jobs.html'
     ordering = ['-id']
 
+    def dispatch(self, *args, **kwargs):
+        # Check if user is authenticated
+        if not self.request.user.is_authenticated:
+            # Redirect them to the home page if not
+            return redirect('home')
+
+        # Render the template if they are
```

Comment:
The code explains its self **in this case**, remove those comments, all over.

Reply:
Fixed.